Last-Update: 2019-08-02
Forwarded: not-needed
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=939891
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: fix FTBFS due to unsupported option.
 Also remove "-Werror" to avoid FTBFS issues due to deprecations when gcc
 is updated.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -206,9 +206,9 @@
 endif()
 
 
 
-set(WB_CXXFLAGS -Werror -Wall -Wextra -Wno-unused-parameter -Wno-deprecated -Wno-deprecated-copy)
+set(WB_CXXFLAGS -Wall -Wextra -Wno-unused-parameter -Wno-deprecated)
 set(WB_CFLAGS -Werror -Wall -Wextra -Wno-unused-parameter -Wno-deprecated -Wno-deprecated-copy)
 
 if(BUILD_FOR_GCOV)
     message ("Enable GCOV due to BUILD_FOR_GCOV")
